assets/blocks/lesson-actions/lesson-actions-block/lesson-actions-editor.scss

```diff
@@ -12,13 +14,30 @@
 		}
 	}
 	&__no-quiz {
-		.wp-block-sensei-lms-button-view-quiz {
-			opacity: 0.1;
+		.wp-block-sensei-lms-button-view-quiz__wrapper {
+			display: none;
 		}
 	}
 	&__complete_lessons-not-allowed {
-		.wp-block-sensei-lms-button-complete-lesson {
-			opacity: 0.1;
+		.wp-block-sensei-lms-button-complete-lesson__wrapper {
+			display: none;
+		}
+	}
+}
+
+.wp-block[data-type="sensei-lms/lesson-actions"] {
+
+	&.is-selected .wp-block-sensei-lms-button-next-lesson__wrapper {
+		position: static;
+		&:before {
+			position: absolute;
+			content: '';
+			left: 12px;
+			right: 12px;
+			border-top: var(--wp-admin-border-width-focus) solid var(--wp-admin-theme-color);
```

Comment:
These variables didn't work for me in the Twenty Twenty One. If there is no common way to use it in all themes I'd suggest using a dashed style with a light gray color. :)

Reply:
It's not the theme, looks like the border-width variable is only in the Gutenberg plugin. Added fallbacks in 6f3a0a6. (The color is for Wordpess 5.4 where this outline was gray)